agent/provider/src/cli/keystore.rs

```diff
@@ -35,6 +38,20 @@ pub struct Add {
         help = "Space separated list of X.509 certificate files (PEM or DER) or PEM certificates chains to be added to the Keystore."
     )]
     certs: Vec<PathBuf>,
+    /// Set certificates permissions for signing certain Golem features.
+    /// If not specified, no permissions will be set for certificate.
+    /// If certificate already existed, permissions will be cleared.
+    #[structopt(
+        short,
```

Comment:
Added here `short` because whitelist cmdline also has `short` params, and these features will be used together.